test.ts

```
@@ -0,0 +1,15 @@
+function greet(name: string) {
+  console.log("Hello, " + name)
+}
+
+let unused = 123
+
+const x = 1
+x = 2
```

Biome (1.9.4)

[error] Can't assign x because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)